doc/src/yosys+odin/user_guide.rst

```diff
@@ -9,11 +9,13 @@ Synthesis Arguments
 
 .. table::
 
-    =======================   ==============================    =================================================================================================
+    =======================   ==============================    =====================================================================================================================================================================
              Arg                    Following Argument                                                          Description
-    =======================   ==============================    =================================================================================================
+    =======================   ==============================    =====================================================================================================================================================================
      `-c`                      XML Configuration File            XML runtime directives for the syntesizer such as the Verilog file, and parametrized synthesis
-     `-V`                      Verilog HDL FIle                  You may specify multiple Verilog HDL files for synthesis									   
+     `-v`                      Verilog HDL File                  You may specify multiple Verilog HDL files for synthesis									   
```

Comment:
specify comma or space separated (or whatever) for multiple file inputs somewhere